examples/worlds/lighter_than_air_blimp.sdf

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" ?>
+<sdf version="1.6">
+  <world name="blimp">
+
+   <physics name="1ms" type="ode">
+      <max_step_size>0.001</max_step_size>
+      <!-- Zero to run as fast as possible -->
+      <real_time_factor>1</real_time_factor>
+    </physics>
+
+    <gravity>0 0 -9.81</gravity>
+
+    <plugin
+      filename="ignition-gazebo-physics-system"
```

Comment:
For forward compatibility use `gz-sim-physics-system` instead.

Reply:
resolved